routes.js

```diff
@@ -0,0 +1,86 @@
+const express = require('express');
+const routes = express.Router();
+
+const db = require('./data/db');
+
+routes.post('/api/posts', async (req,res) => {
+    if (!req.body.title || !req.body.contents) {
```

Comment:
One improvement to save on typing would be to destructure `title` and `contents` from `req.body`.